features/testbot/matching.feature

```diff
@@ -118,6 +118,32 @@ Feature: Basic Map Matching
             | dcba  | hgfe        |
             | efgh  | abcd        |
 
+    Scenario: Testbot - request duration annotations
+        Given the query options
+            | annotations | duration |
+
+        Given the node map
+            """
+            a b c d e   g h
+                i
+            """
+
+        And the ways
+            | nodes    | oneway |
+            | abcdegh  | no     |
+            | ci       | no     |
+
+        And the speed file
+        """
+        1,2,36
+        """
+
+        And the contract extra arguments "--segment-speed-file {speeds_file}"
+
+        When I match I should get
+            | trace | matchings | annotation    |
+            | ach   | ach       | 1,1,0,1,1,2,1 |
```

Comment:
The annotations here are durations, right? How do I read those in the context of the node map? Why is there a duration 0 in there?